### PR TITLE
created a screen

### PR DESCRIPTION
**Pull request by patrycjafilipowicz**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,15 @@
...
 }
+
+.screen {
+    height: 300px;
+    width: 500px;
+    background-color: grey;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    /* bring your own prefixes */
```
Comment (pixellos) [inline src/App.css]: What does that comment meant?

Comment (patrycjafilipowicz) [reply src/App.css]: dear, Mat it means that you can add your own values.

Comment (pixellos) [reply src/App.css]: Please remove the unnecessary comments.

**Comment by pixellos** [comment]
👍 🥇

**Pull request merged**